cdbot/cogs/quoting.py

```diff
@@ -179,9 +179,14 @@ async def multi_quote_embed(self, multi_quote: dict) -> Embed:
         lines = []
         for message in messages:
             line_str = message["user"] + ": "
-            if "```" in message["content"]:
+            if message["content"].startswith("```"):
                 line_str += "\n"
-            line_str += message["content"][:1950]
+            line_str += message["content"]
+            if len(line_str) > 200:
```

Comment:
Why is this 200?

Reply:
At a maximum length of 200, there can be around 10 messages that can fit into a multiquote so I think that'd be a reasonable length to truncate to.

Reply:
Sure. Perhaps that should be moved a constant somewhere then? Is there a check somewhere for the length of the total combined content?

Reply:
I think this was kinda mentioned in the other review you made (the one i still havent resolved), this is one area of the code thats a bit bodged up to make stuff work and could potentially be completely reworked in the future (or now)

Reply:
Ah sorry, hadn't seen the comments you added above. It's worth considering the best way to handle content that exceeds the character limit; at a minimum, there should be a check that truncates the content if it goes over. In more ideal circumstances, some level of pagination could be included, though I don't know how often quotes go over 2000 characters, so I'm not sure if it is worth it if the code is overly complex. Moving numbers to constants for this kind of stuff is probably a good idea regardless.

Reply:
I'll leave this one open incase anyone has any alternate ideas, or if we should go ahead with how it is set up right now.